### PR TITLE
ui: show last updated if untitled doc

### PR DESCRIPTION
**Pull request by meowcodes**
Screenshot 2023-05-12 at 5 16 24 PM

### CODE REVIEW

**Review by jczhong84 — COMMENTED** [review]
why not show it for all datadocs, instead of untitled docs only?

also, can we have a shorter version, e.g.  `20m ago`  `1d ago`, to save more space for longer titles.

**Comment by meowcodes** [comment]
we can do it for other datadocs in another pr if we want.

File: querybook/webapp/components/DataDocNavigator/DataDocGridItem.tsx [hunk 1]
```diff
@@ -45,6 +50,18 @@ export const DataDocGridItem: React.FunctionComponent<IDataDocGridItemProps> =
 
         const { title = '', public: publicDataDoc } = dataDoc;
         const privateIcon = !publicDataDoc ? 'Lock' : null;
+        const todayEpoch = React.useMemo(() => new Date().getTime() / 1000, []);
```
Comment (czgu) [inline querybook/webapp/components/DataDocNavigator/DataDocGridItem.tsx:53]: memo is overkill, just put it in getLastUpdatedDOM

File: querybook/webapp/components/DataDocNavigator/DataDocGridItem.tsx [hunk 2]
```diff
@@ -45,6 +50,18 @@ export const DataDocGridItem: React.FunctionComponent<IDataDocGridItemProps> =
...
+
+        const getLastUpdatedDOM = React.useCallback(
```
Comment (czgu) [inline querybook/webapp/components/DataDocNavigator/DataDocGridItem.tsx:55]: there is no need to put ui rendering code in useCallback.

**Review by czgu — APPROVED** [review]

**Pull request merged**